backend/game/cards/keep_cards/health_manipulation_cards/it_has_a_child.py

```diff
@@ -7,17 +7,9 @@ def __init__(self):
         super().__init__("It Has a Child", 7,
                          "If you reach 0[health] discard all your cards and lose all you [star]. Gain 10[health] and continue playing outside Tokyo.")
 
-    def immediate_effect(self, player_that_bought_the_card, other_players):
-        card = ItHasAChild()
-        player_that_bought_the_card.add_card(card)
-
     def special_effect(self, player_that_bought_the_card, other_players):
         if player_that_bought_the_card.current_health <= 0:
-            self._discard_all_cards(player_that_bought_the_card, other_players)
+            player_that_bought_the_card.discard_all_cards()
```

Comment:
The discard all cards method just clears the list effectively destroying all the cards. The cards should be discarded back to the deckhandler instead

Reply:
Yes realizing this is going to need to be marked as a larger bug as this is how the discard method works for ALL cards currently (both keep and discard)

Reply:
This is marked in #226 and can be handled outside of this change list as it affects much more than just this one card